src/lightning/pytorch/trainer/trainer.py

```diff
@@ -264,6 +264,14 @@ def __init__(
             profiler: To profile individual steps during training and assist in identifying bottlenecks.
                 Default: ``None``.
 
+                .. note::
+                    Do **not** use a manual ``torch.profiler.profile`` context manager around
+                    ``Trainer.fit()``, ``Trainer.validate()``, etc.
+                    This will lead to internal errors and cryptic crashes due to incompatibility between
+                    PyTorch Profiler and Lightning's training loop.
+                    Always use this ``profiler`` argument to enable profiling in Lightning.
+
+
```

Comment:
Overall I think the intention with the rainer arg docstring is that it should be fairly short and on point. There are not a note on any of the other arguments so lets remove this.
```suggestion
```